shacl12-core/index.html

```diff
@@ -3150,14 +4340,50 @@ <h4>sh:languageIn</h4>
 							<div class="turtle">
 ex:Mountain
 	ex:prefLabel "Mountain"@en ;
-	ex:prefLabel "Hill"@en-NZ ;
+	ex:prefLabel "Hill"@en-nz ;
```

Comment:
`en-NZ` is valid. BCP-47 preferred form is `en-NZ`.
Language tags are not required to be lower case.